webapp/first_snap/views.py

```diff
@@ -8,36 +14,107 @@
 )
 
 
-@first_snap.route("/")
-def get_first_snap():
-    return "first snap"
-
-
 @first_snap.route("/<language>")
 def get_language(language):
-    return language
+    context = {"language": language}
+    return flask.render_template(
+        "first-snap/install-snapcraft.html", **context
+    )
+
+
+@first_snap.route("/<language>/<operating_system>/package")
+def get_package(language, operating_system):
+    filename = "".join(["../../first-snap/", language, "/package.yaml"])
```

Comment:
I think you can extract `../../first-snap` into it's own variable in case we want to move it in the future